include/configs/warp7.h

```diff
@@ -45,11 +45,11 @@
 	"mmcargs=setenv bootargs console=${console},${baudrate} " \
 		"root=PARTUUID=${uuid} rootwait rw\0" \
 	"loadbootscript=" \
-		"fatload mmc ${mmcdev}:${mmcpart} ${loadaddr} ${script};\0" \
+		"load mmc ${mmcdev}:1 ${loadaddr} ${script};\0" \
```

Comment:
You did change the mmcpart here and this is not part of this commit.

Reply:
Yes sorry for that.